projects/storefrontapp-e2e-cypress/cypress/helpers/customer-ticketing/customer-ticketing.ts

```diff
@@ -100,7 +107,8 @@ export function clickSubmit(){
 }
 
 export function verifyRequestCompleted(){
-  cy.get('cx-global-message').contains('Request Created');
+  cy.get('cx-global-message').contains('Request created.');
+  cy.get('cx-global-message').contains('Request created.').should('not.exist', { timeout: 10000 });
 }
```

Comment:
Can we do this more generic? To be reusable by other test cases?
Something that allows us to validate the X message is shown.
Maybe pass the text as a parameter.